databricks/koalas/tests/test_dataframe.py

```diff
@@ -3432,6 +3432,7 @@ def test_take(self):
         self.assert_eq(
             kdf.take(range(-1, -3), axis=1).sort_index(),
             pdf.take(range(-1, -3), axis=1).sort_index(),
+            almost=True,
```

Comment:
I made some cases `almost=True` because of various reasons, most of which are related to index/column types.
In this case:

```
E               AssertionError: DataFrame.columns are different
E
E               DataFrame.columns levels are different
E               [left]:  1, Index([], dtype='object')
E               [right]: 2, MultiIndex([], )
```

pandas seems to keep the column's `levels` values and just modify its `codes`.

Another reason is the precision of values, like `mean`, `std`, and `var` aggregations.

We should reduce as many `almost=True` as possible in the future.

Reply:
Oh, yes I got it. Thanks for the explanation :)